internal/db/sqltest/tests/kms/kms_data_key_destruction_job.sql

```diff
@@ -26,14 +26,6 @@ begin;
       ('kdkv_______colors');
   select lives_ok('insert_new_key_id', 'insert of valid key_id in kms_data_key_version_destruction_job failed');
 
-  -- Should fail when inserting an unknown key_id
-  prepare run_insert_unknown_key_id as
-    insert into kms_data_key_version_destruction_job_run
-      (key_id, table_name, total_count)
-    values
-      ('kdkv___unknown', 'auth_token', 100);
-  select throws_ok('run_insert_unknown_key_id', '23503', null, 'insert of unknown key_id in kms_data_key_version_destruction_job_run succeeded');
```

Comment:
It wasn't possible to trigger this _inside_ of a transaction anymore as the consistency check is done at transaction commit time. I couldn't find an easy way to run this test at commit time so I removed it.

Reply:
No, since it's a direct database schema test, we don't really have another place to test this, and frankly I'm not sure it's necessary.